apps/src/Observer.js

```diff
@@ -0,0 +1,52 @@
+/** @file Helper for consuming ObservableEvents. */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+ eqeqeq: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
```

Comment:
Cool. Generally fine with your decisions. Seems a little weird to introduce 90 as maxlen, when our style guide already waffles between 80 and 120. Why not just set maxlen to one of those?